.luacheckrc

```diff
@@ -13,11 +13,13 @@ files["awesomerc.lua"].allow_defined_top = true
 -- This file itself
 files[".luacheckrc"].ignore = {"111", "112", "131"}
 
+-- Theme files, ignore max line length
+files["themes/*"].ignore = {"631"}
```

Comment:
Sorry, I don't understand? Instead of this, you want to add a line to themes that except them from all checks?

Yesterday I learnt that the themes in awesome 3.5 set a global variable `theme` and some people made us of this in their config. This was clearly(?) unintended, so luacheck coverage of themes is a good thing.